include/hyperelasticMaterial.h

```diff
@@ -79,7 +79,7 @@ namespace Solid
     dealii::SymmetricTensor<2, dim>
       b_bar; //!< modified left Cauchy-Green tensor
 
-    /** Return the isochoric part of the Kirchhoff stress. */
```

Comment:
There is a `.clang-format` file which can be used to automatically fix formatting issues like this. Install `clang-format` on your system and run `clang-format -i -style=file <file>`.